src/App.tsx

```diff
@@ -1,61 +1,112 @@
-import './App.scss';
+/*eslint-disable*/
+import "./App.scss";
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from "./api/users";
+import todosFromServer from "./api/todos";
+import { TodoList } from "./components/TodoList";
+import React, { useState } from "react";
+import { Todo } from "./types/ToDo";
+
+const posts = todosFromServer.map((todo) => {
+  const user = usersFromServer.find((person) => person.id === todo.userId);
+  return { ...todo, user };
+});
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(posts);
+  const [titleField, setTitleField] = useState("");
+  const [userName, setUserName] = useState("");
+
+  const [errorTitle, setErrorTitle] = useState(false);
+  const [errorName, setErrorName] = useState(false);
+
+  const nextId = Math.max(...todos.map((todo) => todo.id)) + 1;
```

Comment:
You only use `nextId` inside the `sendPost` function. So it'd be better to move this calculated value there. At the moment on every component re-render you recalculate it which is bad for the performance.